lib/NOM/State.hs

```diff
@@ -194,6 +197,21 @@ data EvalInfo = MkEvalInfo
   }
   deriving stock (Show, Eq, Ord, Generic)
 
+data PrintNameStyle = PrintName | PrintDerivationPath deriving stock (Show, Eq, Ord, Generic)
+
+data PrintState = MkPrintState
+  { printName :: PrintNameStyle
+  , printHelp :: Bool
+  }
+  deriving stock (Show, Eq, Ord, Generic)
+
+initPrintState :: PrintState
+initPrintState =
+  MkPrintState
+    { printName = PrintName
+    , printHelp = False
+    }
+
```

Comment:
I somehow consider this thing more generally "RuntimeConfig" than "PrintState". And I wonder we could even just merge it with "Config". (But that might be annoying depending on where that is used.)